export-api-def/index.js

```diff
@@ -45,5 +45,10 @@ module.exports = yeoman.Base.extend({
     } else {
       process.stdout.write(apiDef);
     }
+
+    // Kill app if still alive
+    if (!app.loaded) {
+      process.exit();
```

Comment:
I remember there were issues with writing to `stdout` and immediately exiting the process afterwards. When the output is longer, then on some platforms like Windows and some Node.js versions, the process will exit before the output was fully printed. See https://github.com/strongloop/loopback-sdk-angular-cli/commit/6a8c379bfc562e8715d3a63288cec29c74116940

While I am not sure if this is still an issue, perhaps we could add a small delay ~30ms before calling `process.exit`, to give the stdout stream a chance to write all data?